libcontainer/rootfs_linux.go

```diff
@@ -40,7 +40,8 @@ type mountConfig struct {
 // mountEntry contains mount data specific to a mount point.
 type mountEntry struct {
 	*configs.Mount
-	srcFD string
+	srcFD   string
+	idmapFd *int
```

Comment:
not very important, but we use `srcFD` so I guess the proper name is `idmapFD`.

Reply:
This was solved already :)